packages/docusaurus-theme-classic/src/theme/BlogPostItem/Header/Author/index.tsx

```diff
@@ -39,7 +107,11 @@ export default function BlogPostItemHeaderAuthor({
               <span>{name}</span>
             </MaybeLink>
           </div>
-          {title && <small className="avatar__subtitle">{title}</small>}
+          {hasSocialMedia ? (
+            renderSocialMedia()
+          ) : (
+            <small className="avatar__subtitle">{title}</small>
+          )}
```

Comment:
I'm not sure we should choose one or the other, but let's discuss and figure this out

Reply:
This remains the main thing we need to solve to unblock this PR.

Can you add many dogfood cases of blog posts with single/many authors, with/without socials? We need to see how it would look like.

It would be preferable to keep displaying the author description, but maybe we can't when there are too many authors? 🤷‍♂️ 

Any multi-author blog inspiration to take a look from?

This Next.js blog only shows basic author info: https://nextjs.org/blog/next-12-3